.azure-pipelines-templates/matrix.yml

```diff
@@ -35,14 +35,13 @@ parameters:
       cmake_args: ''
     debug:
       cmake_args: '-DCMAKE_BUILD_TYPE=Debug -DBUILD_SMALLBANK=OFF'
-    perf:
-      HTTP:
-        cmake_args: ''
-    release:
+    install:
       FTCP:
```

Comment:
Didn't realise I'd left some FTCP stuff in there.

Reply:
Removed now.